lib/Doctrine/ORM/Query/TreeWalkerChainIterator.php

```diff
@@ -106,6 +106,8 @@ public function offsetExists($offset)
     }
 
     /**
+     * @param mixed $offset
```

Comment:
Is this useful?

Reply:
`Mixed` is never good but the function called from there has the same typing so there is not really a different easy fix.
There should be no parameter without either PHP, PHPDpc typing or both. That's the goal of this feature.